src/types/circuit.rs

```diff
@@ -280,7 +281,7 @@ pub fn layout(
             Ok(layout)
         }
         CircuitTypeConcrete::CircuitPartialOutputs(_) => {
-            todo!("CircuitPartialOutputs is noop for now")
+            native_panic!("CircuitPartialOutputs is noop for now");
```

Comment:
This type does have a layout, even though it's a placeholder.
```suggestion
            Ok(Layout::new::<()>())
```